doc/governance/accept_recovery.rst

```diff
@@ -101,33 +135,36 @@ Summary Diagram
 .. mermaid::
 
     sequenceDiagram
-        participant Member 0
-        participant Member 1
-        participant Users
-        participant Node 2
-        participant Node 3
-
-        Note over Node 2, Node 3: Operators have restarted a public-only service
-
-        Member 0->>+Node 2: Propose transition_service_to_open
-        Node 2-->>Member 0: Proposal ID
-        Member 1->>+Node 2: Vote for Proposal ID
-        Node 2-->>Member 1: State: Accepted
-        Note over Node 2, Node 3: transition_service_to_open proposal completes. <br> Service is ready to accept recovery shares.
-
-        Member 0->>+Node 2: GET /gov/encrypted_recovery_share/<member0_id>
-        Node 2-->>Member 0: Encrypted recovery share for Member 0
-        Note over Member 0: Decrypts recovery share
-        Member 0->>+Node 2: POST /gov/recovery_share: "<recovery_share_0>"
-        Node 2-->>Member 0: 1/2 recovery shares successfully submitted.
-
-        Member 1->>+Node 2: GET /gov/encrypted_recovery_share/<member1_id>
-        Node 2-->>Member 1: Encrypted recovery share for Member 1
-        Note over Member 1: Decrypts recovery share
-        Member 1->>+Node 2: POST /gov/recovery_share: "<recovery_share_1>"
-        Node 2-->>Member 1: End of recovery procedure initiated.
-
-        Note over Node 2, Node 3: Reading Private Ledger...
-
-        Note over Node 2: Recovery procedure complete
-        Note over Node 3: Recovery procedure complete
+        participant Member A
+        participant Member B
+        participant Node 0
+        participant Node 1
```

Comment:
We had an unused participant, and weird names for the others. We also showed no communication between the nodes, suggesting some kind of instantaneous process there (at which point they may as well be a single participant?).

Before:
![image](https://github.com/microsoft/CCF/assets/6000239/b57c4962-1eed-4223-bb0c-b548ed720d08)

After:
![image](https://github.com/microsoft/CCF/assets/6000239/edfd32f0-e5ba-496c-b679-e3e3879655d5)

Reply:
I suspect that was copy pasted in, I would just replace the nodes with Network here, I think this diagram is best kept focused on the member interactions only.

Reply:
Sure, that is much neater. Looks like this now:

![image](https://github.com/microsoft/CCF/assets/6000239/bd7d337b-9cbb-4a1c-8876-205b81790577)